<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MOVideoSetting</class>
 <widget class="QDockWidget" name="MOVideoSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>233</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoSetting</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>FillMode</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="m_fillSolid">
         <property name="text">
          <string>&amp;Solid</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_fillWireFrame">
         <property name="text">
          <string>&amp;WireFrame</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_fillPoint">
         <property name="text">
          <string>&amp;Point</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
